nautobot/users/views.py

```diff
@@ -106,12 +105,8 @@ class LogoutView(View):
     """
 
     def get(self, request):
-        logger = logging.getLogger("nautobot.auth.logout")
```

Comment:
In https://docs.nautobot.com/projects/core/en/stable/configuration/optional-settings/#available-loggers there's a note that `nautobot.auth` logger is the logger for authentication events - might be good to use that logger with the new signal handlers for backwards compatibility?

Reply:
Good call outs. Thanks!